### PR TITLE
Little refactoring

### PR DESCRIPTION
**Pull request by isqua**
🐘

### CODE REVIEW

File: assets/js/script.js [hunk 1]
```diff
@@ -1,6 +1,5 @@
 var doc = document;
 var codeInput = doc.querySelector('.gnr-code-input');
-var codeOutput = doc.querySelector('.gnr-code-output');
```
Comment (isqua) [inline assets/js/script.js:3]: It’s not needed to insert data into current document.

File: assets/js/script.js [hunk 2]
```diff
@@ -54,6 +53,7 @@ function setHeadersDefs () {
 //------------------------------
 
 function createTreeFromHTML ( code ) {
+  var codeOutput = document.createElement('div');
```
Comment (isqua) [inline assets/js/script.js:56]: We may create div but not insert it in a document.

File: assets/js/script.js [hunk 3]
```diff
@@ -123,8 +123,7 @@ function makeList ( elem, level ) {
     var classSpan = doc.createElement('span');
     classSpan.classList.add('gnr-elem__class', 'gnr-class');
 
-    for (var i = 0; i < elem.classList.length; i++) {
-      var classItem = elem.classList[ i ];
+    Array.prototype.forEach.call(elem.classList, function (classItem, i) {
```
Comment (isqua) [inline assets/js/script.js:126]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach

File: assets/js/script.js [hunk 4]
```diff
@@ -288,13 +279,7 @@ function showCodeErrors () {
...
 
 function showBemMessage () {
-
-  if ( hasBemWarning ) {
-    bemMessage.classList.remove( 'gnr-hidden' );
-  }
-  else {
-    bemMessage.classList.add( 'gnr-hidden' );
-  }
+  bemMessage.classList.toggle( 'gnr-hidden', ! hasBemWarning );
```
Comment (isqua) [inline assets/js/script.js:282]: https://developer.mozilla.org/en-US/docs/Web/API/Element/classList

File: assets/js/script.js [hunk 5]
```diff
@@ -347,32 +332,21 @@ function checkHeadersLevels () {
...
 //------------------------------
 
 function checkIsSkippedTag ( elem ) {
-
-  if ( skippedTags.indexOf( elem.tagName) >= 0 ) {
-    return true;
-  }
-
-  return false;
+  return skippedTags.indexOf( elem.tagName) >= 0;
```
Comment (isqua) [inline assets/js/script.js:349]: It was like

``` js
if (someExpr) {
  return true;
} else {
  return false;
}
```

now it’s just

``` js
return someExpr;
```

cause of `>=` returns a _Boolean_.

**Comment by yoksel** [comment]
Спасибо : )

**Pull request merged**